homeassistant/components/tplink/entity.py

```diff
@@ -32,13 +32,14 @@ async def _async_wrap(self: _T, *args: _P.args, **kwargs: _P.kwargs) -> None:
 class CoordinatedTPLinkEntity(CoordinatorEntity[TPLinkDataUpdateCoordinator]):
     """Common base class for all coordinated tplink entities."""
 
+    _attr_has_entity_name = True
```

Comment:
Should you set `_attr_name = None` ?

Reply:
Other devices (like sensor) overwrote the name attribute, so if I set it None here, I would expect some issues down the line